tests/system/src/test/java/test/com/sun/javafx/animation/AnimationTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright header should keep 2023, even if it is almost completely changed

Reply:
Even if it wasn't released?

Reply:
Copyrights in source code aren't about whether it is released, so probably best in this case to keep the 2023.